packages/salesforcedx-vscode-core/test/vscode-integration/services/tracking/sourceTrackingService.test.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { SourceTracking } from '@salesforce/source-tracking';
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import { SourceTrackingService } from '../../../../src/services/sourceTrackingService';
+import { testData } from './testdata/sourceTracking';
+
+describe('getSourceStatusSummary', () => {
+  it('Should return a properly formatted string when local and remote changes exist.', async () => {
+    // Arrange
```

Comment:
Just a FYI, I sometimes add the "Arrange, Act, Assert", but I don't think it's a team standard.  Nothing wrong with it being in there, just wanted to give you a heads up that it isn't required.